docs/connect/erlang.md

```diff
@@ -0,0 +1,84 @@
+(connect-erlang)=
+
+# Erlang
+
+:::{div} sd-text-muted
+Connect to CrateDB from Erlang applications.
+:::
+
+## ODBC
+
+:::{rubric} About
+:::
+
+Erlang includes an [ODBC application] out of the box that provides an
+interface to communicate with relational SQL-databases, see also
+[Erlang ODBC examples].
+
+:::{rubric} Synopsis
+:::
+
+`odbcinst.ini`
+```ini
+[PostgreSQL]
+Description     = PostgreSQL ODBC driver
+Driver          = /usr/local/lib/psqlodbcw.so
+```
+```shell
+odbcinst -i -d -f odbcinst.ini
+```
```

Comment:
This section has an overlap with the dedicated ODBC page being added with GH-411. Maybe we can do better to reorganize this, possibly on a subsequent iteration?

Reply:
This has been wrapped into GH-411 per 48777c39.